miniApp/smartNotes/models.py

```diff
@@ -1,3 +1,23 @@
 from django.db import models
 
 # Create your models here.
+class Subject(models.Model):
+    name = models.CharField(max_length=64)
+
+    def __str__(self):
+        return self.name
+
+class Mentor(models.Model):
+    name = models.CharField(max_length=64)
+
+    def __str__(self):
+        return self.name
+
+class SmartNotes(models.Model):
```

Comment:
Why plural name for a model ? when you deal its instance as single note.

Reply:
Did you mean class SmartNotes(models.Model):

Reply:
No, I mean class-name should be singular like `Note`, class name should not be plural like `Notes` or `SmartNotes`.

Reply:
Oh' Got it :) & resolved it as well